### ISSUE TITLE
New red plug symbol suddenly appeared

### ISSUE THREAD

**Issue by permaMonk** [rudders/homebridge-wemo#50]
Since the last Eve app update to vers. 5.4.3 suddenly I see a red plug symbol in the "Power off/on" line. Eve support says it is a warning symbol there is no plug in the Belkin Wemo Insight Switch. But it was and is plugged for years. Eve support says also take the plug out, wait a minute and set it in would help. But it don´t.

To make it clear. Everything works as usual. No problems with the combination Homebridge / wemo plugin / Eve app. Only the red plug symbol appears which give no sense.

Homebridge: 1.4.0
homebridge-wemo: v4.7.1
iOS: 15.3.1

![Image_01](https://user-images.githubusercontent.com/60167885/155885611-d2d8f452-fc73-4039-95e9-8ab7deb2d5b5.jpg)

**Comment by bwp91** [comment]
Hi @permaMonk 

I have never seen this icon before, and I don't totally understand what it means 🤔

**Comment by permaMonk** [comment]
Eve support told me it’s meaning is there is no plug in. I also don’t know how does Eve app detect this. Maybe they set up this behavior with their newest update of the app and set the red plug symbol to on when “In use” is set to “no”? I also have no clue.

**Comment by permaMonk** [comment]
No changes in behavior up to now iOS 15.4.1. Still a red plug symbol although a plug / consumer is in and working. ¯\_(ツ)_/¯

**Comment by bwp91** [comment]
Hey @permaMonk 

Is the icon still there?

**Comment by permaMonk** [comment]
Unfortunately - yes.

—

**Comment by bwp91** [comment]
Without knowing what the icon means it’s hard to tell why it’s there. Have Eve told you what it means?

**Comment by permaMonk** [comment]
Yes. Eve said this symbol comes up if there’s no plug / consumer sticks in.
(in their smart plugs)

—

**Comment by permaMonk** [comment]
I dug out this old thread again, because this red plug symbol and the display "Not in use" bothers me somehow.

I have therefore again contacted the company Belkin and now understand the following a little better. 

Belkin's understanding of "in use" is that "in use" is only displayed over a certain power consumption (adjustable from 2 watts). It has nothing to do with whether or not there is a power plug in it.
 
Eve's understanding has nothing to do with current measurement but only with the fact whether a plug is in there = "in use" or not = "not in use".
 
Now I thought I make it "compatible" if I set the preferences “Power Sensing Settings” to "0" in the Wemo app. But unfortunately this is not possible.

Idea
Is it perhaps possible to set a flag in the HomeBridge Wemo plugin, subsection Wemo Insight Switch, that a plug is inserted?

Thank you for your efforts.

**Comment by bwp91** [comment]
So just to confirm - this red icon appears in Eve when the “outlet in use” setting is OFF rather than ON?

**Comment by bwp91** [comment]
@permaMonk I have just published a beta version with a new option to _force_ enable outlet in use to be true for insight devices. the option should appear under the wemo insights section of the config. i do not own an insight so this has not been tested, but please let me know how you get on and if this _hides_ that red icon in the wemo app 😁

**Comment by permaMonk** [comment]
Not exactly sure I understand the question. But here I send once a screenshot when the plug is on, the power consumption is below 2 watts and therefore the Belkin Wemo Insight Plug means "No consumption".
![IMG_1486](https://user-images.githubusercontent.com/60167885/236725745-62710a90-1ec6-4e6a-bf0a-91427e70010b.jpeg)

 The 2nd image shows when the plug is turned off.
![IMG_1487](https://user-images.githubusercontent.com/60167885/236725766-a08352a6-e100-4a0e-85d0-02f658f8fec0.jpeg)

**Comment by permaMonk** [comment]
Oh thx about the beta. I’ll test today afternoon and will report.

**Comment by permaMonk** [comment]
So I installed the beta "homebridge-wemo v6.0.5-beta.0".

But I can´t find this new option "force enable outlet". You write: "The option should appear under the wemo insights section of the config." Can´t see.

![IMG_1499](https://user-images.githubusercontent.com/60167885/236889759-96a57c0d-2328-4749-9454-2efaf0d80316.jpg)

![IMG_1498](https://user-images.githubusercontent.com/60167885/236889797-1cb6cc75-1c4e-4064-ae59-2080eb5a9be3.jpg)

**Comment by permaMonk** [comment]
Can you give me a little hint?

**Comment by permaMonk** [comment]
Ok ok my fault.
In my preferences I had “Show as” = None. That was wrong obviously. As I corrected this to “Outlet (Default) I could see your new “Outlet In Use - Force True”.
![IMG_1654](https://github.com/bwp91/homebridge-wemo/assets/60167885/579dcf0a-e7b9-48e4-aa67-2e93589a19c7)

With this the red “Nothing plugged symbol” has gone.
![IMG_1653](https://github.com/bwp91/homebridge-wemo/assets/60167885/ef6deee4-6b8f-40ee-9771-82363755d683)
🎉 🎉🎉

So I can say this is working. :)
